### ISSUE TITLE
No Wiki/Issue template

### ISSUE THREAD

**Issue by TheRealBarenziah** [Chevereto/Chevereto-Free#129]
Hello !

It's a suggestion more than an issue, but I believe the (recently born) wiki would gain from having some kind of template/reference page.

It would allow you to only have a single template page to maintain, leaving the job of enforcing the rules to other contributors (that don't even need to be developers).

Also, it would give some guidance to the users, making contributing more enticing.

Issue template have similar benefits but is less important, as long as you're comfortable managing 'em the way they are.

Let me know if I can do anything to help with this - I wrote some basic documentation after managing to run your latest docker image and would gladly put that in your wiki, but being the first entry and having no format guidance is a bit intimidating.

Thanks !

**Comment by rodber** [comment]
Hello @TheRealBarenziah 

I don't use Wikis, I'm a noob in that realm. I don't know how to get started with that neither which templates should be laying around. Sorry about that.

I believe that you opened this issue because of lack of documentation isn't? The documentation at [chevereto/v3-docs](https://github.com/chevereto/v3-docs) covers many concepts that apply for Chevereto-Free and my idea is to fork these docs and implement a stand-alone documentation for Chevereto-Free. It will look like [this](https://v3-docs.chevereto.com).

I expect to proceed with that documenting process once I get to 1.6.

**Comment by TheRealBarenziah** [comment]
> I believe that you opened this issue because of lack of documentation isn't?

I was already aware of the chevereto-v3 docs, and it was useful enough once I got the application running. Still nice of you to link it here  :+1: 

I struggled a bit more with setting up everything in a pure Docker context; the required info is out there but kinda scattered.

As you hinted [here](https://github.com/rodber/chevereto-free/issues/92#issue-946365834) and earlier [in your blog](https://rodolfo.is/2021/06/25/adios-chevereto-free/#sorry) that you'd appreciate some community involvement, I believe a couple of templates/guidelines could go a long way.

> my idea is to fork these docs and implement a stand-alone documentation for Chevereto-Free. It will look like this.
> I expect to proceed with that documenting process once I get to 1.6.

I get that there's already the chevereto forum, the possibility to fork/improve [chevereto/v3-docs](https://github.com/chevereto/v3-docs), and Stack Overflow and issues, so the wiki sounds like just another thing to care of.  
But in this list, only the Github issues are user-friendly in that they don't need another login nor assume PHP knowledge. 

I believe in the end everybody would benefit from you spending less time fixing documentation and more on the interesting stuff !

> I don't use Wikis, I'm a noob in that realm. I don't know how to get started with that neither which templates should be laying around. Sorry about that.

No problemo :smiley: Guess i'll try posting something once I've got my setup right.

In the end this issue is about lacking some 'contributing.md' guidelines to clarify what both PHP and non-PHP people could do to help you with chevereto-free
